components/button.scss

```diff
@@ -224,10 +237,55 @@ $button-font-size: 1rem;
       height: $icon-button-height; 
     }
 
+    &.is-large {
+      @extend %primary-properties;
+      overflow: visible;
+      height: $icon-button-large-height; 
+      width: $icon-button-large-height;
+      font-size: 9px;
```

Comment:
bare 9? og ikke rem?

Reply:
Ser lite ut, men når det er et ord som "Kommentar" så må det være det for at det skal bli plass